winpr/libwinpr/interlocked/test/TestInterlockedAccess.c

```diff
@@ -147,13 +147,13 @@ int TestInterlockedAccess(int argc, char* argv[])
 
 	if (oldValue64 != 0x66778899AABBCCDD)
 	{
-		printf("InterlockedCompareExchange failure: Actual: %lld, Expected: %lld\n", oldValue64, (LONGLONG) 0x66778899AABBCCDD);
+		printf("InterlockedCompareExchange failure: Actual: %ld, Expected: %ld\n", oldValue64, 0x66778899AABBCCDD);
```

Comment:
The format string is wrong, need to define something similar to ```PRId64``` for our own defines. (here it works as long as ```long``` is 8 bytes wide, otherwise ```LONGLONG``` will need ```%lld```)